changelogs/unreleased/gh-3682-uri-encode-decode.md

```diff
@@ -0,0 +1,13 @@
+## feature/lua/uri
```

Comment:
There is quite long story about this bindings...
Please look at discussion here https://github.com/tarantool/tarantool/pull/4153

Reply:
Closed because PR is WIP.